bcipy/helpers/system_utils.py

```diff
@@ -208,21 +208,21 @@ def configure_logger(
     logfile = os.path.join(save_folder, 'logs', log_name)
 
     # configure it
-    root_logger = logging.getLogger()
-    root_logger.setLevel(log_level)
+    custom_logger = logging.getLogger(name=save_folder)
```

Comment:
I think it's good to specify which logger we are configuring, but since you can define more than one handler per logger, consider making the name of the logger different than the save_folder name, which only affects the FileHandler. We may need an additional parameter for that.

`custom_logger = logging.getLogger(name=logger_name)`